DatadogCore/Tests/Datadog/RUM/Integrations/CrashReportReceiverTests.swift

```diff
@@ -993,4 +1129,90 @@ class CrashReportReceiverTests: XCTestCase {
             expectViewURL: RUMOffViewEventsHandlingRule.Constants.backgroundViewURL
         )
     }
+
+    func testGivenCrashDuringAppLaunchWithEventMappers_whenSending_itSendsWithMappedEvents() throws {
```

Comment:
suggestion/ This test feels redundant as we already cover the two control flows that call mappers (the new "with active view" and "without active view" tests). What seems lacking tho is a test that will ensure that view event is not sent when crash (error) is dropped from the mapper. // depends on what we decide for dropping crashes 🙂

Reply:
Replaced this test with one that checks that we don't drop crashes from the mapper.